serverless/installing-serverless-operator.adoc

```diff
@@ -0,0 +1,10 @@
+// Module included in the following assemblies:
+//
+// serverless/installing-openshift-serverless.adoc
+
+= Installing the OpenShift Serverless Operator
+
+The OpenShift Serverless Operator can be installed by using the operator indexed in the OpenShift OperatorHub. 
```

Comment:
s/installed by using the operator indexed in the OpenShift OperatorHub. /can be installed using the {product-title} instructions for installing Operators.

Although I would recommend adding this to the assembly instead so that you can just xref to the whole operators assembly rather than have to inc. modules. The CLI commands for example for installing Operators are not in the module below.

You can see what I mean in the KubeFed docs where I've used this: https://github.com/openshift/openshift-docs/blob/master/kubefed/installing-kubefed.adoc

Reply:
:+1: